code/cow_brd.c

```diff
@@ -417,9 +419,22 @@ static void brd_make_request(struct request_queue *q, struct bio *bio)
     }
     sector += len >> SECTOR_SHIFT;
   }
+#else
```

Comment:
nit: please indent to be even with the code above, use 2 spaces per "tab" please :)